scenarios.md

```diff
@@ -0,0 +1,190 @@
+# Notary Signing - Scenarios
+
+As containers and cloud native artifacts become the common unit of deployment, users want to know the artifacts in their private registries and the artifacts deployed are the same artifacts that were initially published.
+
+The [OCI TOB][oci-tob] has adopted [OCI Artifacts][artifacts-repo], generalizing container images as one of many types of artifacts that may be stored in a registry. Other artifact types currently include:
+
+* [Helm Charts][helm-registry]
+* [Singularity][singularity]
+* Car firmware updates, deployed from OCI Artifact registries
+
+## The Need for a Generalized Signing Solution
+
+This document serves as the requirements and constraints of a generalized signing solution. It focuses on the scenarios and needs, and very specifically avoids any reference to other projects or implementations. As our working group forms a consensus on the requirements, the group will then transition to a spec.
+
+
+## Key Stake Holders & Contributors
+
+As we identify the requirements and constraints, a number of key contributors will be asked to represent their requirements and constraints.
+
+> Please add companies, projects, products that you believe should be included.
+
+* Registry Cloud Operators
+  * [Azure Container Registry (acr)][acr] - Steve Lasker <steve.lasker@microsoft.com> (@stevelasker)
+  * [AWS Container Registry (ecr)][ecr] - Omar Paul <omarpaul@amazon.com>
+  * [Docker Hub][docker-hub]
+  * [Google Container Registry (gcr)][gcr]
+  * [GitHub Package Registry (gpr)][gpr]
+  * [Quay][quay]
+  * [IBM Cloud Container Registry (icr)][icr]
+* Registry Vendors, Projects & Products
+  * [Docker Trusted Registry][docker-dtr]
+  * [Harbor][harbor]
+  * [JFrog Artifactory][jfrog]
+* Artifact Types
+  * Container Images - OCI Org
+  * Helm Charts
+  * Singularity
+  * Operator Bundles
+
+## Scenarios
+
+### Scenario #1: Local Build, Sign, Validate
+
+Prior to doing any deployment, a developer can test the: build, sign, validate scenario.
+
+1. Locally build a container image using a non-registry specific `name:tag`, such as:  
+  `$docker build hello-world:1`
+1. Locally sign `hello-world:1` 
```

Comment:
I believe we ended with:
We'll focus on the scenarios here, and work with each specific implementation to support the scenario. For eg: Justin mentioned Derek has PRs to merge efforts into the ContainerD work, where this would be enabled.